tests/gamma/csm_observability_test.py

```diff
@@ -32,6 +32,7 @@
 from framework.test_app.runners.k8s import gamma_server_runner
 from framework.test_app.runners.k8s import k8s_base_runner
 from framework.test_app.runners.k8s import k8s_xds_client_runner
+from framework.test_app.runners.k8s import k8s_xds_server_runner
```

Comment:
```suggestion
```

This should be removed now too - now it's unused import

Reply:
Done. Good catch. Thanks